helpdesk_mgmt/controllers/main.py

```diff
@@ -12,8 +14,64 @@ def support_ticket_close(self, **kw):
         values = {}
         for field_name, field_value in kw.items():
             values[field_name] = field_value
-        ticket = http.request.env['helpdesk.ticket'].sudo().\
-            search([('id', '=', values['ticket_id'])])
-        ticket.stage_id = values.get('stage_id')
+        ticket = http.request.env['helpdesk.ticket'].search([(
+            'id', '=', values['ticket_id'])])
+        stage_id = int(values.get('stage_id'))
+        ticket.sudo().update(
```

Comment:
`ticket.stage_id = ...` after putting the sudo before

Reply:
Sorry but I don't understand what you mean.

Reply:
That you don't need to sudo again. `ticket` is already "sudoed" thanks to previous browse with sudo.